src/Pos/Communication/Limits.hs

```diff
@@ -277,6 +278,11 @@ instance MessageLimited (MsgHeaders ssc) where
         return $
             MsgHeaders <$> vectorOf Const.recoveryHeadersMessage headerLimit
 
+instance MessageLimitedPure MsgSubscribe where
+    msgLenLimit = 0
```

Comment:
This will almost certainly also change with the CBOR patch. It'll probably be 1 or 2.